src/Migrator/index.ts

```diff
@@ -369,25 +381,40 @@ export class Migrator extends EventEmitter implements MigratorContract {
 	/**
 	 * Migrate down (aka rollback)
 	 */
-	private async runDown(batch?: number) {
+	private async runDown(batch?: number, step?: number) {
```

Comment:
Should I suggest something here?

Instead of accepting two numbers that cannot exist together. We should accept a number and its type. Something like

```ts
runDown(strategy: 'batch' | 'step', value: number)
```

Inside the command itself we can raise an exception if `--batch` and `--step` is used. So that those flags are not leaking into the core implementation.

Agree?